lib/offsite_payments/integrations/directebanking.rb

```diff
@@ -89,7 +89,13 @@ def initialize(order, account, options = {})
 
         # Need to format the amount to have 2 decimal places
         def amount=(money)
-          cents = money.respond_to?(:cents) ? money.cents : money
+          cents = if money.respond_to?(:cents)
+            money.cents
+          elsif money.respond_to?(:subunits)
+            money.subunits
+          else
+            money
+          end
```

Comment:
Could you refactor that so the logic is in one place ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself))? Maybe a convenience method?

Convenience method:
```ruby
module MoneyCompatibility
  def to_cents(money)
    if money.respond_to?(:cents)
      money.cents
    elsif money.respond_to?(:subunits)
      money.subunits
    else
      money
    end
  end
end
```

Since [try](https://api.rubyonrails.org/classes/Object.html#method-i-try) is available you could also just do:
```ruby
  money.try(:cents) || money.try(:subunits) || money
```